### PR TITLE
Azurestack Aks admin commands

### PR DESCRIPTION
**Pull request by Bhuvaneswari-Santharam**
Azurestack AKS Admin commands - initial version

### CODE REVIEW

**Comment by Bhuvaneswari-Santharam** [comment]
@bganapa - Could you please review?

**Comment by bganapa** [comment]
@Bhuvaneswari-Santharam  Need pester tests as we discussed

File: src/Azs.ContainerService.Admin/test/loadEnv.ps1 [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# ----------------------------------------------------------------------------------
```
Comment (bganapa) [inline src/Azs.ContainerService.Admin/test/loadEnv.ps1:1]: this file should not be needed to be checked in

File: src/Azs.ContainerService.Admin/docs/Get-AzsContainerServiceQuota.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+```
+
+## DESCRIPTION
+Returns a list of container service quotas at the given location.
+
+## EXAMPLES
+
+### -------------------------- EXAMPLE 1 --------------------------
+```powershell
+{{ Add code here }}
```
Comment (bganapa) [inline src/Azs.ContainerService.Admin/docs/Get-AzsContainerServiceQuota.md:27]: did you run generate-help ?

**Review by bganapa — APPROVED** [review]

**Pull request merged**